Python_Scripts/sbml2amici.py

```diff
@@ -7,16 +7,22 @@
 import pandas as pd
 
 from C import (
-    DIR_BASE, DIR_MODELS_SEDML, DIR_MODELS_AMICI_BASE, DIR_MODELS_AMICI)
+    DIR_BASE, DIR_MODELS_REGROUPED, DIR_MODELS, DIR_MODELS_AMICI)
 
 # create .tsv file
-tsv_table = pd.DataFrame(
-    columns=['id', 'states', 'reactions', 'error_message'])
+model_info = pd.read_csv(os.path.join(DIR_MODELS, 'model_summary.tsv'),
```

Comment:
I just hate the pandas api. Dunno whether it would work the way you suggested. If so, cool, will change. But it's just so counter-intuitive when which shorthands work in pandas. All sadists, those folks

Reply:
I better not tag the pandas github repo now here ;) then they would be notified :rofl:

Reply:
It's not like I haven't already complained there... E.g., `DataFrame.append()` simply doesn't append in the python sense, and you can't even make it append by specifying optional inputs...